qos-service-api/build.gradle

```diff
@@ -12,5 +12,12 @@ libsDirName = file('build/artifacts')
 dependencies {
     compile group: 'javax.ws.rs', name: 'javax.ws.rs-api'
     compile group: 'com.palantir.safe-logging', name: 'safe-logging'
+    compile group: 'com.fasterxml.jackson.core', name: 'jackson-databind'
+    compile group: 'javax.validation', name: 'validation-api'
```

Comment:
Is this really a compile dep for API?

Reply:
Was thinking about adding the `@Size(min = 1)` for clientLimits, but decided to go without it as the service will not be able to startup without configuring it if I add the check.